Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -173,6 +171,8 @@ + (void)removePasteboardWithName:(NSString*)pasteboardName {
 /**
  @Status Caveat
  @Notes only NSString, NSURL and UIImage are supported.
+  If any of supported type in pasteboardTypes is supported, return YES.
+  otherwise, return NO. (reference platform also has the same behavior)
```

Comment:
true. removed. the behavior was a surprise to me 

---
In reply to: [87639485](https://github.com/Microsoft/WinObjC/pull/1358#discussion_r87639485) [](ancestors = 87639485)